test/e2e/util.go

```diff
@@ -1627,6 +1627,194 @@ func createGenericJobAWWithStatus(context *context, name string) *arbv1.AppWrapp
 	return appwrapper
 }
 
+func createGenericJobAWWithMultipleStatus(context *context, name string) *arbv1.AppWrapper {
+	rb := []byte(`{
+		"apiVersion": "batch/v1",
+		"kind": "Job",
+		"metadata": {
+			"name": "aw-test-job-with-comp-ms-21",
```

Comment:
I think we should use the name passed as an argument here and throughout the test, and then append to that name the 1 and 2 that separate the two generic items.

Reply:
Done, thanks